.packit.yml

```diff
@@ -3,8 +3,6 @@ actions:
     # this is being triggered immediately on a pull_request event; wait for
     # build-dist.yml to generate the dist tarball.  git rebase needs an
     # identity in order to rewrite the committer field.
-    - git config --global user.name 'nobody'
-    - git config --global user.email '<>'
```

Comment:
Oops! Thanks for fixing that. I certainly ahve ran `packit srpm` a lot of times locally, curiously it didn't destroy my `~/.gitconfig`.